client/Cargo.toml

```diff
@@ -22,7 +22,8 @@ zeroize_derive = "1.0"
 anyhow = "1.0"
 thiserror = "1.0"
 futures = "0.3"
-riker = "0.4"
+riker = { version = "0.4", git ="https://github.com/elenaf9/riker", branch = "master" }
```

Comment:
Curious as to why you are pointing it to your own repo.  Is there some unmerged changed that needs to be here for the client and for the comms layer?

Reply:
```
where
    Req: MessageEvent,
    T: Message + From<Req>,
{
   pub client: ActorRef<T>
```
This trait bound depends on a [PR at riker](https://github.com/riker-rs/riker/pull/162) that has been approved, but somehow not merged yet(it seems like the maintainer of the repo is currently inactive). In my fork I already merged the PR, which is why is used it as dependency. 
If this is an issue, I could change the trait restriction to sth like this (which would be possible without the PR merged):
```
where
    Req: MessageEvent,
    T: Tell<Req> + Clone + Debug + ... ,
{
   pub client: T
```

Reply:
Alright, that's what I suspected.  Its fine as it is then.